core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -599,6 +601,7 @@ object KafkaConfig {
     "implement the <code>org.apache.kafka.server.policy.CreateTopicPolicy</code> interface."
   val AlterConfigPolicyClassNameDoc = "The alter configs policy class that should be used for validation. The class should " +
     "implement the <code>org.apache.kafka.server.policy.AlterConfigPolicy</code> interface."
+  val LogMessageDownConversionEnableDoc = "This configuration controls whether down-conversion of message formats is enabled to satisfy consume requests."
```

Comment:
Seems not ideal to have this documentation in two places. I know you are just following the current pattern, but if we don't actually have any difference to communicate, perhaps we should just refer to the documentation in `TopicConfig`?